css-transforms-2/Overview.bs

```diff
@@ -1106,6 +1106,65 @@ The transform functions <<matrix()>>, ''matrix3d()'' and ''perspective()'' get c
 
 For interpolations with the primitive ''rotate3d()'', the direction vectors of the transform functions get normalized first. If the normalized vectors are equal, the rotation angle gets interpolated numerically. Otherwise the transform functions get converted into 4x4 matrices first and interpolated as defined in section <a href="#matrix-interpolation">Interpolation of Matrices</a> afterwards.
 
+Addition and accumulation of transform lists {#combining-transform-lists}
+============================================
+
+<a lt="value addition">Addition</a> of transform lists is
+defined as <a spec=infra>list</a> concatenation
+such that <var ignore>V<sub>result</sub></var> is equal to
+<var ignore>V<sub>b</sub></var> <a spec=infra for=list>appended</a> to
+<var ignore>V<sub>a</sub></var>.
+
+<a lt="value accumulation">Accumulation</a> of transform lists
+follows the same steps as interpolation
+with regards to matching transform functions including
+padding lists with <a>identity transform functions</a>,
+converting ''transform/none'' to an <a>identity transform function</a>,
+and converting both arguments to matrices as necessary (see
+[[css-transforms-1#interpolation-of-transforms]]).
+However, instead of interpolating the individual parameters,
+they are combined using arithmetic addition--
+except in the case of parameters whose value is one in the
+<a>identity transform function</a>
+(e.g. scale parameters and matrix elements <var ignore>m11</var>, <var
+ignore>m22</var>, <var ignore>m33</var>, and <var ignore>m44</var>),
+which combine using <dfn>accumulation for one-based values</dfn> as follows:
+
+  <var ignore>V<sub>result</sub></var> =
+  <var ignore>V<sub>a</sub></var> + <var ignore>V<sub>b</sub></var> - 1
+
+<div class="note">
+The above definition preserves the intent of <a lt="value
```

Comment:
Please indent the contents of block elements. (I think the rest of the spec is inconsistent in this regard, but it's better to be correct here than consistent.)